packages/wallets/src/metamask/pages/elements/header.element.ts

```diff
@@ -11,7 +13,8 @@ export class Header {
 
   constructor(page: Page) {
     this.page = page;
-    this.networkSetting = new NetworkSetting(page);
+    this.networkList = new NetworkList(this.page);
+    this.networkSetting = new NetworkSetting(this.page);
```

Comment:
i think, network list and network seeting its a same classes

Reply:
I thought about it. But the NetworkSetting has many locators of the network edit modal, and I think it is correct to  separate it the another class.